### PR TITLE
Updated archetypes to bootstrap 3.3.0

### PR DESCRIPTION
**Pull request by svenkubiak**
Updated to bootstrap 3.3.0, styled system messages and changed html5shim to https.

### CODE REVIEW

File: ninja-core/src/site/markdown/developer/changelog.md [hunk 1]
```diff
@@ -1,28 +1,8 @@
-Version X.X.X
+Version xxx
 =============
 
- * 2015-01-03 Bump to doctester 1.1.6 (sparkoo)
```
Comment (raphaelbauer) [inline ninja-core/src/site/markdown/developer/changelog.md:4]: That deletion does not look right... Or did I miss something?

**Comment by raphaelbauer** [comment]
nice! there is just one comment...

**Comment by svenkubiak** [comment]
The deletion must have happened when i last fetched the upstream. Sorry for that. 
I bumped the versions again, as they were already outdated. Also, I fixed the "human error" in the changelog ;-)

**Comment by svenkubiak** [comment]
Hm, don't know will Buildhive failes. 4.1.17-1 is the latest version of tiny-mce....

**Comment by raphaelbauer** [comment]
nice :)

**Comment by raphaelbauer** [comment]
my local build is also complaining:

[INFO] [ERROR] Failed to execute goal on project basic: Could not resolve dependencies for project archetype.it:basic:war:0.1-SNAPSHOT: Could not find artifact org.webjars:tinymce-jquery:jar:4.1.17-1 in central (https://repo.maven.apache.org/maven2) -> [Help 1]
[INFO] org.apache.maven.lifecycle.LifecycleExecutionException: Failed to execute goal on project basic: Could not resolve dependencies for project archetype.it:basic:war:0.1-SNAPSHOT: Could not find artifact org.webjars:tinymce-jquery:jar:4.1.17-1 in central (https://repo.maven.apache.org/maven2)

**Comment by svenkubiak** [comment]
I've reverted to tiny-mce to 4.0.16, which seems to work.

**Comment by raphaelbauer** [comment]
nice!

**Pull request merged**